manifests/0000_90_kube-scheduler-operator_03_servicemonitor.yaml

```diff
@@ -50,3 +50,12 @@ spec:
       for: 15m
       labels:
         severity: critical
+  - name: scheduler-legacy-policy-deprecated
+    rules:
+    - alert: SchedulerLegacyPolicySet
+      annotations:
+        message: The scheduler is currently configured to use a legacy scheduler policy API. Use of the policy API is deprecated and removed in 4.10.
+      expr: |
+        cluster_legacy_scheduler_policy > 0
+      labels:
+        severity: warning
```

Comment:
Usually we wait for 60 min for warning level alert - https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#warning-alerts, is it consistent with rest of deprecation alerts?

Reply:
@ingvagabund yeah, that's a fair argument, would you mind opening a followup?

Reply:
Here: https://github.com/openshift/cluster-kube-scheduler-operator/pull/363